CHANGELOG.md

```diff
@@ -5,51 +5,61 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - 2024-08-08
+
+### Added
+
+-   Initial release of the Python script to generate a Task Scheduler XML file.
```

Comment:
Let's try to reduce this to a single line, or nest sub-features as bullets beneath main bullet (like how it was done in 0.1.0). IMO these are implementation details that aren't necessary.